release.sh

```diff
@@ -104,6 +104,7 @@ export KO_DOCKER_REPO="gcr.io/knative-nightly"
 # Build stripped binary to reduce size
 export GOFLAGS="-ldflags=-s -ldflags=-w"
 export GITHUB_TOKEN=""
+export TAG TAG_RELEASE
```

Comment:
Can you just add `export` to line 85 and 88?

Reply:
In this case, yes, I can.

I often write it like that to not fall onto [SC2155](https://github.com/koalaman/shellcheck/wiki/SC2155).